modules/developer-cli-odo-converting-an-s2i-component-into-a-devfile-component.adoc

```diff
@@ -0,0 +1,37 @@
+[id="converting-an-s2i-component-into-a-devfile-component_{context}"]
+= Converting an S2I component into a devfile component
+
+With `{odo-title}`, you can create both S2I and devfile components. If you have an existing S2I component, you can convert it into a devfile component using the `odo utils` command. 
+
+.Procedure
+
+Run all the commands from the S2I component directory.
+
+. Run the `odo utils convert-to-devfile` command which creates `devfile.yaml` and `env.yaml` based on your component:
```

Comment:
s/command which/command, which/

Reply:
s/creates `devfile.yaml` and `env.yaml` based/creates the `devfile.yaml` and `env.yaml` files based/